Core/include/Acts/Seeding/SeedfinderUtils.hpp

```diff
@@ -0,0 +1,240 @@
+#pragma once
```

Comment:
The file will need an ACTS header.

Also, it will have to apply with ACTS Coding standards, which is CamelCase.